package.json

```diff
@@ -3,12 +3,13 @@
   "license": "LGPL-3.0-or-later",
   "scripts": {
     "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle",
-    "watch": "watch -p 'static/sass/**/*' -p 'static/js/**/*.js' -c 'yarn run build'",
+    "watch": "watch -p 'static/sass/**/*' -p 'static/js/**/*.js' -p 'static/js/**/*.ts' -c 'yarn run build'",
     "watch-js": "webpack --watch",
     "watch-scss": "watch -p 'static/sass/**/*.scss' -c 'yarn run build-css'",
     "build": "yarn run build-css && yarn run build-js",
     "build-css": "sass static/sass/styles.scss static/css/styles.css --load-path=node_modules --style=compressed && postcss --map false --use autoprefixer --replace 'static/css/**/*.css'",
-    "build-js": "yarn run build-js-bundle",
+    "type-check-js": "tsc",
+    "build-js": "yarn run build-js-bundle && yarn run type-check-js",
```

Comment:
Shouldn't you type check first?

Reply:
I think that's a good idea. Will do the type check before I bundle.